types/primitive.yaml

```diff
@@ -123,9 +123,21 @@ LogsBloom:
   example: "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
   pattern: "^0x[a-fA-F0-9]{512}$"
 
+Blob:
+  type: string
+  format: hex
+  pattern: "^0x[a-fA-F0-9]{262144}$"
+  description: "A blob is `FIELD_ELEMENTS_PER_BLOB * size_of(BLSFieldElement) = 4096 * 32 = 131072` bytes (`DATA`) representing a SSZ-encoded Blob as defined in Deneb"
+
 KZGCommitment:
   type: string
   format: hex
   pattern: "^0x[a-fA-F0-9]{96}$"
   description: "A G1 curve point. Same as BLS standard \"is valid pubkey\" check but also allows `0x00..00` for point-at-infinity"
   example: "0x93247f2209abcacf57b75a51dafae777f9dd38bc7053d1af526f220a7489a6d3a2753e5f3e8b1cfe39b56f43611df74a"
+
+KZGProof:
+  type: string
+  format: hex
+  pattern: "^0x[a-fA-F0-9]{96}$"
+  description: "A KZG proof. Same check as `KZGCommitment`"
```

Comment:
Would be better to make this description explicit rather than refer to another type.

Reply:
Updated in https://github.com/ethereum/beacon-APIs/pull/286/commits/4ccaaa6f018c596c6ebf1227943da952ed7cf29f, thanks!